src/nodes.coffee

```diff
@@ -2509,7 +2509,8 @@ exports.Op = class Op extends Base
     return @compileUnary        o if @isUnary()
     return @compileChain        o if isChain
     switch @operator
-      when '?'  then @compileExistence o
+      when '?'  then @compileExistence o      # Check if not null and not undefined
+      when '??' then @compileExistence o, yes # Check only if not undefined
```

Comment:
Yeh... Again I still think we're making a mistake by moving to ES' default semantics, and should have stuck with `null`/`undefined` being treated the same in CS. So I'm probably not the best person to ask 😛